homeassistant/components/aurora_abb_powerone/config_flow.py

```diff
@@ -88,8 +75,13 @@ async def async_step_user(
 
         errors = {}
         if self._com_ports_list is None:
-            result = await self.hass.async_add_executor_job(scan_comports)
-            self._com_ports_list, self._default_com_port = result
+            self._com_ports_list = await self.hass.async_add_executor_job(
+                usb.list_serial_ports
+            )
```

Comment:
It might make sense to hide the executor call behind a `usb.async_list_serial_ports`